zaza/model.py

```diff
@@ -876,6 +876,17 @@ def check_unit_workload_status_message(model, unit, message=None,
         raise ValueError("Must be called with message or prefixes")
 
 
+def check_applications_status(app_data, status):
+    """Check that the applications in the model matches the supplied status.
+
+    :param model: Application data
+    :type model: dict
+    :param status: Acceptable application status
+    :type status: list
+    """
+    return app_data.status in status
```

Comment:
It feels a bit like this function might be a bit of overkill.  It's value might be if the underlying structure in `app_data` is changing frequently and thus `.status` might be moving around or changing type, etc, but it seems that `some_app_data.status in some_status_condition_line` is easier to type (and potentially understand) than `check_applications_status(some_app_data, some_status_condition_line)`?